src/Chinchilla/Subscription.cs

```diff
@@ -65,6 +65,12 @@ public IWorkersController Workers
 
         public void Start()
         {
+            if (IsStarted)
+            {
+                logger.InfoFormat("Subscription already started: {0}", this);
```

Comment:
@thingylab so the reason why Start is public is because there are scenarios where your subscription isn't automatically started, this can happen when your delivery strategy is not startable... e.g. you are using a worker pool delivery strategy with no initial workers assigned to process messages.